modules/perception/inference/tensorrt/plugins/leakyReLU_plugin.h

```diff
@@ -0,0 +1,80 @@
+/******************************************************************************
+ * Copyright 2018 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#pragma once
+
+#include <algorithm>
+#include <vector>
+
+#include "modules/perception/inference/tensorrt/rt_common.h"
+
+namespace apollo {
+namespace perception {
+namespace inference {
+
+class ReLUPlugin : public nvinfer1::IPlugin {
+ public:
+  ReLUPlugin(const ReLUParameter &param, const nvinfer1::Dims &in_dims) {
+    input_dims_.nbDims = in_dims.nbDims;
+    CHECK_GT(input_dims_.nbDims, 0);
+    for (int i = 0; i < in_dims.nbDims; i++) {
+      input_dims_.d[i] = in_dims.d[i];
+      input_dims_.type[i] = in_dims.type[i];
+    }
+    negative_slope_ = param.negative_slope();
+  }
+
+  ReLUPlugin() {}
+  ~ReLUPlugin() {}
+  virtual int initialize() { return 0; }
```

Comment:
CamelCase.

Reply:
you mean `initialize()`? I can't change this name, it is required by the TensorRT library.

Reply:
Got it. Thanks!